docs/functions/functions.json

```diff
@@ -1424,6 +1572,69 @@
 				true
 			]
 		},
+		{
+			"name": "filename",
+			"id": "filename",
+			"unary": false,
+			"category": "other",
+			"role": "function",
+			"argument": [
+				{
+					"name": "URL",
+					"optional": false,
+					"type": "Text"
+				}
+			],
+			"description": "Gets all texts after the rightmost slash of an URL. For example, `filename(\"http://mavo.io/docs/functions.html\")` will return `functions.html`.\nIf the parameter is not an URL, `filename()` will return the value of the parameter as-is.",
+			"returnType": "Text",
+			"returnDescription": "Text after the rightmost slash of an URL. Returns the value of the parameter as-is if not an URL."
+		},
+		{
+			"name": "json",
+			"id": "json",
+			"unary": false,
+			"category": "other",
+			"role": "function",
+			"description": "Converts data into JSON string and avoids circular reference.",
```

Comment:
The part about circular references is confusing to novices, this is not something they should worry about, I'd suggest removing it.
Also, what's a "JSON string"? You mean a string of JSON, but also any quoted and properly escaped text is technically a "JSON string". Just say it converts data to JSON, that's perfectly sufficient and not ambiguous!
Same applies for all your other mentions of "JSON string". JSON is a text-based format by itself, you don't need to add "string" after it. That's like documenting any function that produces HTML as producing an "HTML string".